aci/data_source_aci_bfdmhifp.go

```diff
@@ -0,0 +1,60 @@
+package aci
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ciscoecosystem/aci-go-client/v2/client"
+	"github.com/ciscoecosystem/aci-go-client/v2/models"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceAciBfdMultihopInterfaceProfile() *schema.Resource {
+	return &schema.Resource{
+		ReadContext:   dataSourceAciBfdMultihopInterfaceProfileRead,
+		SchemaVersion: 1,
+		Schema: AppendBaseAttrSchema(AppendNameAliasAttrSchema(map[string]*schema.Schema{
+			"logical_interface_profile_dn": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"key": &schema.Schema{
```

Comment:
`key` is not set when we do the important and data source apply operation. Will it be the expected behaviour?

`key` is marked as a `Sensitive` attribute in the resource. Is it required for data source?

Reply:
key can be removed. so removed